extensions/raindrop-io/package.json

```diff
@@ -8,11 +8,7 @@
   "author": "lardissone",
   "contributors": [
     "shubham",
-    "pernielsentikaer",
-    "sh-cho",
-    "tdkn",
-    "j3lte",
-    "rien7"
+    "pernielsentikaer"
```

Comment:
Why is there SO many changes and removed contributors here?

Reply:
hmm, weird, I didn't changed this. I think the pull-contributors command broke it.

Let me restore these changes.

Reply:
Thanks 🙂

Reply:
Will close this PR and create a new one. Something was wrong with my outdated fork.